settings/pkg/assets/assets.go

```diff
@@ -27,7 +26,7 @@ type assets struct {
 func (a assets) Open(original string) (http.File, error) {
 	if a.config.Asset.Path != "" {
 		if stat, err := os.Stat(a.config.Asset.Path); err == nil && stat.IsDir() {
-			custom := filepath.Join(
+			custom := path.Join(
```

Comment:
To be honest I don't get what difference this makes on a linux or mac computer. We are trying to use `filepath` everywhere because it is able to handle backslashes. Could you elaborate why/how this fixes asset loading? And for what it's worth, it might deserve a changelog then. ;-)

Reply:
my fault, i had some strange errors yesterday. it was on my machine only. closed